ukrainian/call-a-doctor/doctor.md

```diff
@@ -0,0 +1,5 @@
+З'являється лікар і пропонує Вам з'їсти синю або червону зефірку.
+
+[З'їм синю зефірку, просто випустіть мене звідси!](call-a-doctor/pills/blue-pill.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.